firefox_puppeteer/ui/toolbars.py

```diff
@@ -229,15 +244,11 @@ def value(self):
         return self.urlbar.get_attribute('value')
 
 
-class AutocompleteResults(BaseLib):
+class AutocompleteResults(UIBaseLib):
     """Wraps DOM elements and methods for interacting with autocomplete results."""
 
-    def __init__(self, *args, **kwargs):
-        BaseLib.__init__(self, *args, **kwargs)
-        # TODO: A "utility" module that sets up the client directly would be
-        # useful here.
-        self.l10n = L10n(self.get_marionette)
-        self.keys = Keys(self.get_marionette)
+    def __init__(self, marionette_getter, window):
+        UIBaseLib.__init__(self, marionette_getter, window)
```

Comment:
If there is no custom initialization code you can simply remove this method. Python is smart enough to call `__init_` from the super class automatically.